ace-extensions.d.ts

```diff
@@ -5,3 +5,17 @@ declare module "ace-code/src/ext/simple_tokenizer" {
     }>>
     export function tokenize(content: string, highlightRules: import("./ace").Ace.HighlightRules): TokenizeResult;
 }
+
+declare module "ace-code/src/ext/modelist" {
+    export type Mode = {
+        name: string;
+        caption: string;
+        mode: string;
+        extensions: string;
+        extRe: RegExp;
```

Comment:
I think we could treat `extRe` as an internal property of the `Mode` (and maybe update implementation to have it as `_extRe`), since it's not used anywhere outside the `Mode` class and I think it shouldn't be - there is `supportsFile` method for it.

Reply:
So I would suggest to treat it as internal and not expose it in a type.

Reply:
Sounds good, done in 3bb0bc2.